components/web-components/toggle/index.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react'
+
+import * as S from './style'
+
+export type Size = 'sm' | 'default'
+export type Brand = 'vpn' | 'shields'
+interface ToggleProps {
+  size?: Size
+  isOn?: boolean
+  disabled?: boolean
+  accessibleLabel?: string
+  onChange?: (isOn?: boolean) => unknown
```

Comment:
nit: `isOn` isn't optional, it will always be a `boolean`, right?

Reply:
this one is used for default value upon component render

Reply:
when `onChange` is called it's always passed a boolean isn't it? Are we talking about the same thing?